babel.config.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+
+module.exports = {
+  presets: [[require.resolve('@babel/preset-react'), { runtime: 'automatic' }]],
+
+  env: {
+    production: {
+      presets: [
+        [
+          require.resolve('@babel/preset-env'),
+          {
+            modules: 'commonjs',
+            targets: { browsers: ['last 1 Chrome version'] },
+          },
+        ],
+      ],
+    },
+
+    development: {
+      presets: [
+        [
+          require.resolve('@babel/preset-env'),
+          {
+            modules: 'commonjs',
+            targets: { browsers: ['last 1 Chrome version'] },
+          },
+        ],
+      ],
+      plugins: [require.resolve('react-refresh/babel')],
+    },
+
+    test: {
+      presets: [
+        [
+          require.resolve('@babel/preset-env'),
+          {
+            modules: 'commonjs',
+            targets: { node: 'current' },
+          },
+        ],
+      ],
+      plugins: [
+        [
+          require.resolve('babel-plugin-istanbul'),
```

Comment:
can remove this plugin (it is needed for code coverage when doing cypress e2e/component tests)

Reply:
removed